utils/p4runtime_lib/helper.py

```diff
@@ -110,8 +110,8 @@ def get_match_field_pb(self, table_name, match_field_name, value):
             lpm.mask = encode(value[1], bitwidth)
         elif match_type == p4info_pb2.MatchField.RANGE:
             lpm = p4runtime_match.range
```

Comment:
not introduced by this PR, but the variable name (`lpm`) doesn't make sense for ternary and range matches